Sources/JavaKitMacros/JavaClassMacro.swift

```diff
@@ -87,13 +87,21 @@ extension JavaClassMacro: MemberMacro {
       }
       """
 
+    let nonOptionalAs: DeclSyntax = """
+      /// It's not checking anything.
```

Comment:
This isn't quite enough.

This should document why the conversion will never fail and the implementation should do so accordingly.

```suggestion
    // Casting to <. the target type .> will never be nil because <. the java type .> <<extends it || or some other reason>>.


```